packages/tools/components-package/wdio.js

```diff
@@ -57,7 +57,7 @@ exports.config = {
 			// to run chrome headless the following flags are required
 			// (see https://developers.google.com/web/updates/2017/04/headless-chrome)
 			args: ['--headless', '--disable-gpu'],
-			//args: ['--disable-gpu'],
```

Comment:
Why this?

Reply:
VS Code comment feature, will be removed.